x-pack/plugin/mapper-version/src/yamlRestTest/resources/rest-api-spec/test/50_script_values.yml

```diff
@@ -40,6 +42,70 @@ setup:
                 source: "field('ver').get(new Version(''))"
 
   - match: { hits.hits.0.fields.field.0: "5.4.3" }
-  - match: { hits.hits.1.fields.field.0: "1.2.3.4.5" }
-  - match: { hits.hits.2.fields.field.0: "1.2.3-abc+def" }
-  - match: { hits.hits.3.fields.field.0: "1.0.0" }
+  - match: { hits.hits.1.fields.field.0: "1.12.3" }
+  - match: { hits.hits.2.fields.field.0: "1.2.3.4.5" }
+  - match: { hits.hits.3.fields.field.0: "1.2.3-abc+def" }
+  - match: { hits.hits.4.fields.field.0: "1.0.0" }
+
+
+---
+"Sort by Version script value (asc)":
```

Comment:
This will fail on other versions, you'll have to move this into a new yaml test with the setup
```
  - skip:
      version: " - 8.2.99"
      reason: "version script field sorting support was added in 8.3.0"
``